src/Ledger/Conway/Specification/Utxo/Properties.agda

```diff
@@ -0,0 +1,9 @@
+{-# OPTIONS --safe #-}
+
+module Ledger.Conway.Specification.Utxo.Properties where
+
+open import Ledger.Conway.Specification.Utxo.Properties.Computational
+open import Ledger.Conway.Specification.Utxo.Properties.DepositHelpers
```

Comment:
I would not include this a as "Properties" file. (Since it is auxiliary functions)